src/app/routes.tsx

```diff
@@ -189,6 +190,15 @@ const diagnosticsRoutes: IAppRoute[] = [
     navSubgroup: ANALYZE,
     featureLevel: FeatureLevel.BETA,
   },
+  {
+    component: HeapDumps,
+    label: 'Heap Dumps',
+    path: toPath('/heapdumps'),
+    title: 'Heap Dumps',
+    description: 'Create and view heap dumps on single target JVMs.',
+    navGroup: DIAGNOSTICS,
+    navSubgroup: ANALYZE,
```

Comment:
This should have `FeatureLevel.BETA` like Thread Dumps has, at least for now. Once these features are cleaned up, and in particular once there is an All Targets view that allows users to clean up data from lost targets, then we can remove the feature flagging and promote these to production grade features.